migrations/versions/407e74de5553_adding_user_table.py

```diff
@@ -8,14 +8,13 @@
 
 # revision identifiers, used by Alembic.
 revision = '407e74de5553'
-down_revision = '3a5eba38e4e8'
+down_revision = '12f9dcc2e36c'
```

Comment:
This looks pretty evil to me.  This rolls back a *long* way and misses a whole bunch of migrations that happened in-between. I think the migration sequence should be linear and not potentially branch like this.